Library/Homebrew/vendor/README.md

```diff
@@ -1,7 +1,9 @@
 Vendored Dependencies
 =====================
 
-* [okjson](https://github.com/kr/okjson), version 43.
+* [okjson](https://github.com/kr/okjson), version 43
+
+* [plist](https://github.com/bleything/plist), version unknown
```

Comment:
Glad to have this but the `version unknown` worries me a bit. Could we maybe try to bump it to a known revision at some point?